elide-datastore/elide-datastore-hibernate/src/main/java/com/yahoo/elide/core/hibernate/hql/RootCollectionFetchQueryBuilder.java

```diff
@@ -61,7 +61,7 @@ public Query build() {
                         + SPACE
                         + filterClause
                         + SPACE
-                        + getSortClause(sorting, entityClass, false)
+                        + getSortClause(sorting, entityClass, true)
```

Comment:
The only problem with true/false parameters, you have to check the doc on the method to see what this means.  But that is not new to this PR.